Server/ConcordiaCurriculumManager/Services/CourseService.cs

```diff
@@ -147,17 +136,35 @@ public async Task<CourseModificationRequest> InitiateCourseModification(CourseMo
         {
             _logger.LogWarning($"Error retrieving the dossier ${typeof(Dossier)} ${dossier.Id}: does not belong to user ${typeof(User)} ${userId}");
             throw new Exception("Error retrieving the dossier: does not belong to the user");
-        }
-
-        bool courseModified = await _courseRepository.UpdateCourse(course);
-        if (!courseModified)
+        }
+
+        var newModifiedCourse = new Course
+        {
+            Id = new Guid(),
+            CourseID = course.CourseID,
+            Subject = modification.Subject,
+            Catalog = modification.Catalog,
+            Title = modification.Title,
+            Description = modification.Description,
+            CreditValue = modification.CreditValue,
+            PreReqs = modification.PreReqs,
+            Career = modification.Career,
+            EquivalentCourses = modification.EquivalentCourses,
+            CourseState = CourseStateEnum.CourseChangeProposal,
+            Version = 1,
```

Comment:
Can you set version to course.Version, so it can have same version as course we're modifying (when we publish the change, that's when version will be implemented)

Reply:
Done